frontend/src/Editor/QueryManager/QueryEditors/Stripe.jsx

```diff
@@ -135,22 +136,44 @@ class StripeComponent extends React.Component {
   };
 
   computeOperationSelectionOptions = (operationOptions) => {
-    let options = [];
+    let pgs = []; // array of objects
     const paths = operationOptions?.paths;
 
     if (paths) {
       for (const path of Object.keys(paths)) {
         for (const operation of Object.keys(paths[path])) {
-          options.push({
-            value: `${operation},${path}`,
-            name: path,
-            operation: operation,
-          });
+          //path looks like get/v1/account/{id}
+          let interim = path.split('/')[2]; // gives category of path, here "account"
+          let found = pgs.findIndex((obj) => obj.label === interim); //returns index of object that has label key's value as interim
+
+          if (found >= 0) {
+            // object with label as interim exists in pgs
+            pgs[found]['options'].push({
+              value: `${operation}${path}`,
```

Comment:
I think it will be better to revise to not use flags but to show the actual intent of categorising in the code instead of comments. Wdyt?
```js
  computeOperationSelectionOptions = (operationOptions) => {
    const paths = operationOptions.paths;
    if (isEmpty(paths)) return [];

    const selectionOptions = Object.keys(paths).reduce((acc, path) => {
      const operations = Object.keys(paths[path]);
      const category = path.split('/')[2] || ' Default' ; // the assumption is that there will always be /api/*, maybe we need a default category?

      operations.forEach((operation) => {
        const option = {
          value: `${operation}${path}`,
          label: `${operation}${path}`,
          name: path,
          operation: operation,
        };

        const existingCategory = acc.find((obj) => obj.label === category);

        if (existingCategory) {
          existingCategory.options.push(option);
        } else {
          acc.push({
            label: category,
            options: [option],
          });
        }
      });

      return acc;
    }, []);

    return selectionOptions;
  };
  ```

Reply:
We ae appending operation to get its value inside changeOperation function for the operation select dropdown.

`const changeOperation = (value) => {
    const operation = value.split('/', 2)[0];
    const path = value.substring(value.indexOf('/'));
    const newOptions = { ...options, path, operation, selectedOperation: specJson.paths[path][operation] };
    setOptions(newOptions);
    props.optionsChanged(newOptions);
  };`